schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -1442,12 +1442,12 @@ qudt:Unit
   rdfs:subClassOf [
       a owl:Restriction ;
       owl:allValuesFrom qudt:Unit ;
-      owl:onProperty qudt:hasDenominatorPart ;
+      owl:onProperty qudt:qkdvDenominatorPart ;
     ] ;
   rdfs:subClassOf [
       a owl:Restriction ;
       owl:allValuesFrom qudt:Unit ;
-      owl:onProperty qudt:hasNumeratorPart ;
+      owl:onProperty qudt:qkdvNumeratorPart ;
```

Comment:
Do we need to append 'Part'?

Reply:
I thought about whether we should just call the property qkdvNumerator, without "Part", and decided to leave it in. But I could be convinced if you would like it removed. Also open to any other naming suggestion, before we start heavily using it.

Regarding pointing to a unit, I don't think so. Unit points to QuantityKind, which in turn points to QuantityKindDimensionVector.  (Jack was only temporarily linking some of the new IEC units to the dimension vectors in his branch).